systemtests/src/test/java/io/enmasse/systemtest/bases/web/WebConsoleTest.java

```diff
@@ -74,6 +74,28 @@ protected void doTestCreateDeleteAddress(Destination... destinations) throws Exc
         assertWaitForValue(0, () -> consoleWebPage.getResultsCount(), new TimeoutBudget(20, TimeUnit.SECONDS));
     }
 
+    protected void doTestCreateDeleteDurableSubscription(Destination... destinations) throws Exception {
+        consoleWebPage = new ConsoleWebPage(selenium, getConsoleRoute(sharedAddressSpace), addressApiClient,
+                sharedAddressSpace, defaultCredentials);
+        consoleWebPage.openWebConsolePage();
+        consoleWebPage.openAddressesPageWebConsole();
+
+        for (Destination dest : destinations) {
+            //create topic
+            consoleWebPage.createAddressWebConsole(dest);
+
+            //create subscription
+            Destination subscription = Destination.subscription(dest.getAddress() + "-subscriber");
```

Comment:
I would create a subscription destination with an address and the topic specified separately. E.g. 

```
Destination subscription = Destination.subscription(dest.getAddress() + "-subscriber", dest.getAddress(), "standard-subscription");
```

Then in the createAddressWebConsole() method (as commented above) you can use the topic directly.